docs/clusters/how-to-set-up-codec-server.md

```diff
@@ -0,0 +1,48 @@
+---
+id: how-to-set-up-codec-server
+title: How to set up a Codec Server
+sidebar_label: Codec Server
+description: Implementation of a remote encryption/decryption server.
+---
+
+The [Codec Server Go sample](https://github.com/temporalio/samples-go/tree/main/codec-server) is an example that shows how to decode a Payload that has been encoded so the Payload can be displayed by [tctl](/tctl) and the [Web UI](/web-ui/how-to-use-a-list-filter-in-the-temporal-web-ui).
+
+A codec HTTP protocol specifies two endpoints to handle Payload encoding and decoding.
+
+Implementations must do the following:
+
+- Send and receive Payloads protobuf as JSON.
+- Check only the final part of the incoming URL to determine whether the request is for /encode or /decode.
+
+Each platform has a different approach to setting up the endpoints for the Codec Server. Follow these steps to get a Codec Server running on your preferred platform.
+
+:::note
+A Temporal Cluster should already be in operation before starting the Codec Server.
+:::
+
+## tctl
+
+[Start up the Codec Server](https://github.com/temporalio/samples-go/tree/main/codec-server).
+
+Pass the encoded Payloads to the codec endpoint:
```

Comment:
The command doesn't pass the Payload, it just configures the endpoint that tctl will use to pass the Payload.

On first read - it suggests the former.